inputs/uprev_rdg_storage_format_version.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
I guess some people like bash, but I prefer python with `import sh`.  I want to minimize the amount of sh code in our codebase.

Reply:
I'll add a TODO to consider reworking this to python